KsApi/models/Config.swift

```diff
@@ -2,6 +2,18 @@ import Argo
 import Curry
 import Runes
 
+public enum Experiment {
+
+  public enum Name: String {
+    case iosTest = "ios_test_test"
```

Comment:
Is `ios_test_test` the name being returned from production at this stage?

Reply:
Yes, but I think it will be removed as soon as we implement real tests.